### ISSUE TITLE
Odd bug with 2.10.4/1.0-M2

### ISSUE THREAD

**Issue by fancellu** [scala-blitz/scala-blitz#45]
With this gist

https://gist.github.com/fancellu/a90941a2b316dd344648

I get odd compiler error

Description Resource    Path    Location    Type
value sum is not a member of scala.collection.par.Par[List[Double]] playpen     Unknown Scala Problem

Scala 2.10.4

```
<dependency>
    <groupId>com.github.scala-blitz</groupId>
    <artifactId>scala-blitz_2.10</artifactId>
    <version>1.0-M2</version>
</dependency>
```

Doesn't show this for 2.11.1 and
<dependency>
        <groupId>com.github.scala-blitz</groupId>
        <artifactId>scala-blitz_2.11</artifactId>
        <version>1.1</version>
    </dependency>

**Comment by fancellu** [comment]
I'm not that concerned, just wanted to try with 2.10 as I had a 2.10 project I was working on. But 2.11 is obviously the future.

**Comment by DarkDimius** [comment]
should be fixed with #44 in latest release
